src/QMCWaveFunctions/RotatedSPOs.h

```diff
@@ -165,15 +165,10 @@ class RotatedSPOs : public SPOSet
 
   void checkInVariables(opt_variables_type& active) override
   {
-    //reset parameters to zero after coefficient matrix has been updated
-    for (int k = 0; k < myVars.size(); ++k)
-      myVars[k] = 0.0;
-
     if (Optimizable)
     {
       if (myVars.size())
```

Comment:
This function needs documentation. You're going to have to write it.  What is it doing?  Explain why its now ok that it doesn't call storeParams (does it also get called only once in the state cycle and that's before the RotatedSPOSet constructor, that seems impossible.